tests/foreman/cli/test_user.py

```diff
@@ -376,9 +366,8 @@ def test_positive_create_ssh_key_super_admin_from_file(self, ssh_key):
         :CaseImportance: Critical
         """
         ssh_name = gen_string('alpha')
-        with get_connection() as connection:
-            result = connection.run(f'''echo '{ssh_key}' > test_key.pub''')
-        assert result.return_code == 0, 'key file not created'
+        result = default_sat.execute(f"echo '{ssh_key}' > test_key.pub")
```

Comment:
@JacobCallahan I think this won't work because we deleted `ssh_key` fixture. We need to use `ssh_key = gen_ssh_keypairs()[1]` from `robottelo.helpers`. Similar to what you have done in `tests/foreman/api/test_user.py`.

Reply:
ahh good catch!